client/src/components/add-employer-component/addEmployer.jsx

```diff
@@ -0,0 +1,100 @@
+import { useState } from "react";
+import AddCompanyInfo from "./company-info-component";
+import AddEmployerInfo from "./employer-contact-component";
+import AddEmployerJobLead from "./job-lead-component";
+
+function AddEmployer() {
+  const [page, setPage] = useState(1);
+  const [showAddSecondaryButton, setShowAddSecondaryButton] = useState(true);
+  const initialState = {
+    companyInfo: [
+      {
+        id: 0,
+        businessName: "",
+        businessLegalName: "",
+        naicsCode: "",
+        phoneNumber: "",
+        faxNumber: "",
+        generalEmail: "",
+        website: "",
+        employerAddress: null,
+        city: null,
+        province: "",
+        postalCode: "",
+      },
+    ],
+    employerContacts: [
+      {
+        id: 0,
+        name: "",
+        jobTitle: "",
+        phoneNumber: "",
+        email: "",
+        alternatePhoneNumber: "",
+      },
+    ],
+    jobLeads: [
+      {
+        id: 0,
+        jobTitle: "",
+        compensation: "",
+        hoursPerWeek: "",
+        description: "",
+        creationDate: null,
+        expirationDate: null,
+        employmentType: "",
+      },
+    ],
+  };
+  const [employerData, setEmployerData] = useState(initialState);
+
+  const handlePageChange = (newPage) => {
+    setPage(newPage);
+  };
+
+  const updateEmployerData = (section, data) => {
+    setEmployerData((prevData) => ({
+      ...prevData,
+      [section]: data,
+    }));
+  };
+
+  const resetEmployerData = () => {
```

Comment:
No need to reset the state here. We'll navigate back to the dashboard (will be handled in a future pr) which should result in us losing the state here anyways. For now you can leave this method empty